### DISCUSSION TITLE
I can't sign in

### DISCUSSION THREAD

**Discussion by Yassineg80** [community/community#118978] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I cannot sign into github because both of my emails are not receiving the authentication code for my passwords. My friends from the same University are able to receive emails from Github so I don't believe it is a domain error. But my gmail is also not able to receive the authentication. I have check junk email and all my inboxes. I am just not receiving it,.

**Comment by kknankk** [comment]
I have the same problem. When I sign up on another device, I can never receive the verification emails. 
I tried to register a new account, but cannot receive emails either. I asked my roommate to try and he also failed. 
I think we need to contact github's staff to fix this.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Yassineg80 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
